cob/config/ompl_planning.yaml

```diff
@@ -82,6 +82,21 @@ arm_right:
     - PRMstarkConfigDefault
   projection_evaluator: joints(arm_right_1_joint,arm_right_2_joint)
   longest_valid_segment_fraction: 0.01
```

Comment:
I just noticed this, but you might want to tune the segment fraction down to 0.005 like the other robots, seems to be the better choice for robots at this scale.